packages/web/src/components/EventsList/index.tsx

```diff
@@ -0,0 +1,2 @@
+export * from './EventsList';
+export * from './utils';
```

Comment:
This shouldn't be necessary outside this component; if it is, let's move the relevant pieces to a separate file that's more globally accessible.

```suggestion
```